primitives/core/src/assertion.rs

```diff
@@ -298,7 +300,6 @@ impl Assertion {
 			// BRC20 Holder
 			Self::BRC20AmountHolder => vec![Web3Network::BitcoinP2tr],
```

Comment:
how to you think to add back the other four btc types here, and do a combination of filtering of `input network type` against `vc supported type` ?